src/index.ts

```diff
@@ -147,11 +147,14 @@ export class SolidLogic {
       })
   }
 
-  load (doc: NamedNode | NamedNode[] | string) {
+  load (doc: NamedNode | NamedNode[] | string, options?: any) {
     if (!this.store.fetcher) {
       throw new Error('Cannot load doc(s), have no fetcher')
     }
-    return this.store.fetcher.load(doc)
+    // force = true to reload the doc
+    // but first remove doc: this is to avoid just adding quads
+    if (options && options.force) this.store.removeDocument(doc as NamedNode)
```

Comment:
I'm quite new to Typescript so may have misunderstood but if someone passed a NamedNode array or a string to load() will it work to pass that to removeDocument as a NamedNode?

Reply:
Ii means that if you use `force = true` doc must be a Named node.
I took this solution because it was used elsewhere in the code, it was simple, it was compatible with the used cases and offer a plus possibility to `load().` In an other way there is no regression

If you don't use force = true then load can receive NamedNode, NamedNode Array or String like before.

Reply:
@edwardsph I made the changes. I don't know anything in typescript. My initial idea was wrong. Load now can pass a NamedNode, NamedNode Array or a string to load() and pass that to removeDocument() as namedNode.

Tested locally with all 3 types.